config/owasp/suppressions.xml

```diff
@@ -22,4 +13,14 @@
     <packageUrl regex="true">^pkg:maven/org\.springframework\.security/spring\-security\-crypto@.*$</packageUrl>
     <cve>CVE-2018-1258</cve>
   </suppress>
+  <suppress until="2020-01-31">
+    <notes><![CDATA[No fix available. file name: tomcat-embed-websocket-9.0.29.jar]]></notes>
+    <packageUrl regex="true">^pkg:maven/org\.apache\.tomcat\.embed/tomcat\-embed\-websocket@.*$</packageUrl>
+    <cve>CVE-2019-17563</cve>
+  </suppress>
+  <suppress until="2020-01-31">
+    <notes><![CDATA[No fix available. file name: tomcat-embed-core-9.0.29.jar]]></notes>
+    <packageUrl regex="true">^pkg:maven/org\.apache\.tomcat\.embed/tomcat\-embed\-core@.*$</packageUrl>
+    <cve>CVE-2019-17563</cve>
+  </suppress>
```

Comment:
or alternatively we could bump to 9.0.30 which fixes `CVE-2019-17563` ;)

Reply:
good suggestion 👍 done